fuel-indexer-tests/assets/fuel_indexer_test.graphql

```diff
@@ -31,7 +31,6 @@ type PungEntity {
 type Block {
     id: Bytes32! @unique
     height: UInt8!
-    producer: Address!
```

Comment:
- So I understand the API has changed and `producer` is no longer available here. 
- Did we remove the concept of a `producer` from fuel-core? 
- This an important small piece of the block explorer demo (knowing which entity produced which block) so we should add this back if at all possible

Reply:
Correct, `producer` has removed from the block object. In my initial investigation, I didn't see it defined anywhere else, but I'll do another check and see if we can get it back.